src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -200,6 +226,43 @@ const PlrDaoStakingTransactionBlock = ({
     resetTransactionBlockFieldValidationError(transactionBlockId, 'toAssetAddress');
   }, [selectedFromNetwork]);
 
+  const updateAvailableRoutes = useCallback(
+    debounce(async () => {
+      setSelectedRoute(null);
+      setAvailableRoutes([]);
+      if (!sdk || !selectedFromAsset || !amount || !selectedFromNetwork?.chainId || !isValidAmount(amount)) return;
+
+      if (receiverAddress && !isValidEthereumAddress(receiverAddress)) {
+        setTransactionBlockFieldValidationError(transactionBlockId, 'receiverAddress', 'Invalid receiver address');
+        return;
+      }
+
+      setIsLoadingAvailableRoutes(true);
+
+      try {
+        const { items: routes } = await sdk.getAdvanceRoutesLiFi({
+          fromChainId: selectedFromNetwork.chainId,
+          toChainId: plrDaoAsset.chainId,
+          fromAmount: ethers.utils.parseUnits(amount, selectedFromAsset.decimals),
+          fromTokenAddress: selectedFromAsset.address,
+          toTokenAddress: plrDaoAsset.address,
+          toAddress: receiverAddress ?? undefined,
+        });
+        setAvailableRoutes(routes);
+        if (routes.length === 1) setSelectedRoute(mapRouteToOption(routes[0]));
```

Comment:
One question in case we don't have the routes length as 1 how's the scenario handled?